### PR TITLE
OM-865 | Removed box-shadows.

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Removed box-shadow element from containers.

### CODE REVIEW

File: src/common/profileSection/profileSection.module.css [hunk 1]
```diff
@@ -3,13 +3,11 @@
   --title-fontsize: var(--fontsize-h-3);
   background: var(--color-white);
   border-radius: 2px;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
   padding: var(--spacing-l) var(--horizontal-whitespace);
   margin-bottom: var(--spacing-xl);
 }
 
 .profileSectionTitleRow {
-  align-items: baseline;
```
Comment (Jokinen) [inline src/common/profileSection/profileSection.module.css:12]: I assume that this was by intention, but I'm commenting to highlight just to be sure.

**Review by Jokinen — APPROVED** [review]
👍

File: src/common/profileSection/profileSection.module.css (hunk 1 above)
Comment (tuovinensanttu) [reply src/common/profileSection/profileSection.module.css:12]: Yeah it was. `profileSectionTitleRow` had align-items set twice.

**Review by badrange — APPROVED** [review]
👍

**Pull request merged**